docs/SAFE_MODE.md

```diff
@@ -0,0 +1,61 @@
+# Safe Mode
+
+**⚠️ Safe mode is experimental and does not provide any guarantees of safety or security.**
+
+Open Interpreter is working on providing an experimental safety toolkit to help you feel more confident running the code generated by Open Interpreter.
+
+
+Install Open Interpreter with the safety toolkit dependencies as part of the bundle:
+
+```shell
+pip install open-interpreter[safe]
```

Comment:
We can leverage the `tool.poetry.extras` configuration to allow folks to optionally install `semgrep` and other safety tools via the `[safe]` extra.

Reply:
YES. I love this idea. 

I think it would be great to make a slimmed down version of OI called `[core]` (wouldn't even include `rich` or any terminal interface), `[safe]` for semgrep, perhaps docker, etc, then eventually `[all]` if we build features beyond safe mode with dependencies.